data/compliance_profiles/schema.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Why is the schema in the Puppet module's `data/` directory?

Unless Hiera uses this file for APL, I'd expect it to live in a somewhere else, like `schemas/` or maybe `utils/schemas/`.